tests/tracer/test_writer.py

```diff
@@ -202,7 +202,7 @@ def test_drop_reason_buffer_full(self):
         statsd = mock.Mock()
         writer_metrics_reset = mock.Mock()
         with override_global_config(dict(health_metrics_enabled=False)):
-            writer = AgentWriter(agent_url="http://asdf:1234", buffer_size=5235, dogstatsd=statsd)
```

Comment:
https://github.com/DataDog/dd-trace-py/pull/5326/commits/a4136ae4447aaa797efcd34510b023a61cf90c03 changes the size of the span object which causes the following [test failure](https://circleci.com/gh/DataDog/dd-trace-py/2139627). This fix is required to resolve the failing test.